.jshintrc

```diff
@@ -115,6 +115,9 @@
         // Underscore.js globals
         "_",
 
+        // Backbone.js globals
+        "Backbone",
```

Comment:
I tried this, and it works. The problem is, the 'common' js tests are not currently configured to use requirejs, and I get `TypeError: require is not a function` when I try to run them. I have tried including requirejs in the karma config, but I can't get it to work. I have reverted the change for now, I will try again if I have time at the end of the sprint.